diagnostic.md

```diff
@@ -6,70 +6,90 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
```

Comment:
We were looking for 
```
get '/books/:id', to: 'books#show'
get '/books/:id' => 'books#show'
```

Specifically, telling rails where to route the GET request to `/books/4`